governance/interface.go

```diff
@@ -107,6 +107,7 @@ type blockChain interface {
 	blockchain.ChainContext
 
 	CurrentHeader() *types.Header
+	CurrentBlock() *types.Block
```

Comment:
How about removing `CurrentHeader()`? It's not used by governance

Reply:
Thanks! I removed `CurrentHeader` and `GetBlockByNumber`